src/scriptworker/log.py

```diff
@@ -57,6 +57,12 @@ def update_logging_config(context: Any, log_name: Optional[str] = None, file_nam
         # If we rotate the log file via logrotate.d, let's watch the file
         # so we can automatically close/reopen on move.
         handler = logging.handlers.WatchedFileHandler(path)  # type: ignore
+    elif context.config.get("log_max_bytes", 0) or context.config.get("log_max_backups", 0):
```

Comment:
maybe `and` instead of `or` ?

Reply:
Mmmm I think we should go by `log_max_bytes` only then.
Wouldn't make sense to have unlimited max_bytes but max_backups > 0

Reply:
Does this mean you want to allow for max_backups == 0 but we blow away the log every log_max_bytes?

Reply:
If so, I'd lean towards the default of max_backups to be, say, 10 in constants.py.